.gitignore

```diff
@@ -1,3 +1,4 @@
+/impl/target
```

Comment:
Is it possible to figure out what command you ran that created this? The parent directory is a workspace root so Cargo should never do anything with impl/target.

Reply:
I think it was `rls`.

Reply:
Could you please revert this file, make a minimal repro and file an rls bug? I would prefer not to check in the workaround.

Reply:
Done.